static/build/.tmp_update/script/miyoogamelist_gen.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+rootdir="/mnt/SDCARD/Roms"
+out='miyoogamelist.xml'
+
+generate_miyoogamelist() {
+    # create backup of previous miyoogamelist.xml
+    if [ -f "$out" ]; then
+        mv "$out" "$out.bak"
+    fi
+
+    echo '<?xml version="1.0"?>' >$out
+    echo '<gameList>' >>$out
+
+    for rom in *; do
+        # ignore subfolders because miyoogamelist don't work with them
+        # also ignores Imgs folder, nice
+        if [ -d "$rom" ]; then
+            continue
+        fi
+
+        # .xml, .bak & .db files are no games
+        # .bin files are no games, .cue files are!
+        if echo "$rom" | grep -qE "\.(xml|bak|db|bin)$"; then
+            continue
+        fi
+
+        filename="${rom%.*}"
+        digest=$(echo "$rom" | grep -oE "^([^(]*)" | awk '{$1=$1};1')
+
+        cat <<EOF >>$out
+    <game>
+        <path>./$rom</path>
+        <name>$digest</name>
+        <image>./Imgs/$filename.png</image>
```

Comment:
Imgs path should also come from the emu config, since some systems doesn't use "./Imgs" (PICO for example)

Reply:
indeed, this is now done